Modules/IxNetRestApiPacketCapture.py

```diff
@@ -185,11 +183,11 @@ def packetCaptureGetCurrentPackets(self, getUpToPacketNumber=20, capturePacketsT
                             with open(packetCaptureFilenameControl, 'a') as packetCaptureFile:
                                 packetCaptureFile.write('\nStack: %s\n' % displayName)
 
-                    response = self.ixnObj.get(self.ixnObj.httpHeader+stackIdObject+'/field', silentMode=False)
-                    for eachField in response.json():
-                        fieldId = eachField['id']
-                        fieldName = eachField['displayName']
-                        fieldValue = eachField['fieldValue']
+
+                    for eachFieldObj in eachStack.Field.find():
```

Comment:
have to give only one line of space if  you want to differentiate..don't give too many line spaces..please use linters